### PR TITLE
replace cherrypy.request.user

### PR DESCRIPTION
**Pull request by ticoann**
related to #8868

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan please review

**Comment by ticoann** [comment]
test please

**Comment by ticoann** [comment]
test this please

File: src/python/WMCore/ReqMgr/Service/Request.py [hunk 1]
```diff
@@ -463,7 +465,7 @@ def _handleOnlyStateTransition(self, workload, request_args, dn):
         return report
 
     def _updateRequest(self, workload, request_args):
-        dn = cherrypy.request.user.get("dn", "unknown")
+        dn = get_user_info("dn", "unknown")
```
Comment (amaltaro) [inline src/python/WMCore/ReqMgr/Service/Request.py]: `dn = get_user_info().get("dn", "unknown")`

**Review by amaltaro — CHANGES_REQUESTED** [review]
I made a comment in the code that has to be updated. I guess that's why that unit test keeps failing

**Comment by ticoann** [comment]
Thanks Alan, I updated the patch.

**Comment by ticoann** [comment]
Error is very weird. I wonder why it didn't fail before. (hope second commit fix that problem)

**Comment by ticoann** [comment]
Alan if this looks OK, please included in the new tag. If not we can push this later.

**Comment by amaltaro** [comment]
Thanks!

**Pull request merged**